### ISSUE TITLE
Accounts/Users search doesn't work. 

### ISSUE THREAD

**Issue by rymorgan** [reactioncommerce/reaction#2802]
### Expected behavior
Search for orders by email, order id etc should get results

### Actual Behavior
Search doesn't work for orders or accounts
Receive this error in the browser console
```
modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:84822 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Check the render method of `SearchModal`.
    in SearchModal (created by SearchModalContainer)
    in div (created by SearchModalContainer)
    in div (created by SearchModalContainer)
    in SearchModalContainer (created by Tracker(SearchModalContainer))
    in Tracker(SearchModalContainer)
<snip>

 Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Check the render method of `SearchModal`.
    at invariant (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:84983:15)
    at instantiateReactComponent (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:106914:56)
    at instantiateChild (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:106722:28)
    at http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:106749:16
    at traverseAllChildrenImpl (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:108378:5)
    at traverseAllChildren (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:108473:10)
    at Object.instantiateChildren (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:106748:7)
    at ReactDOMComponent._reconcilerInstantiateChildren (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:106300:41)
    at ReactDOMComponent.mountChildren (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:106339:27)
    at ReactDOMComponent._createInitialChildren (http://localhost:3000/packages/modules.js?hash=c97b01d6a009e6e565aca5e788a2d3f551872d8c:103550:32)
```

### Steps to Reproduce the Behavior
Place some orders
Create Users
Login as admin
Search for email addresses 
Observe it doesn't work.

**Comment by rymorgan** [comment]
I'm getting this error:
`modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:85619 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Check the render method of `SearchModal`.
    in SearchModal (created by SearchModalContainer)
    in div (created by SearchModalContainer)
    in div (created by SearchModalContainer)
    in SearchModalContainer (created by Tracker(SearchModalContainer))
    in Tracker(SearchModalContainer)`

**Comment by rymorgan** [comment]
And this error: 
`Uncaught Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in. Check the render method of `SearchModal`.
    at invariant (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:85779)
    at instantiateReactComponent (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:107807)
    at instantiateChild (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:107615)
    at modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:107642
    at traverseAllChildrenImpl (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:109271)
    at traverseAllChildren (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:109366)
    at Object.instantiateChildren (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:107641)
    at ReactDOMComponent._reconcilerInstantiateChildren (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:107193)
    at ReactDOMComponent.mountChildren (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:107232)
    at ReactDOMComponent._createInitialChildren (modules.js?hash=b6e103d47151e09be8285e30953956cc18c1b826:104443)`

**Comment by spencern** [comment]
resolved by #2834
